packages/tools/locales/vi.po

```diff
@@ -7,22 +7,26 @@ msgid ""
 msgstr ""
 "Project-Id-Version: Joplin-CLI 1.0.0\n"
 "Report-Msgid-Bugs-To: \n"
-"Last-Translator: \n"
+"Last-Translator: PHUCBM\n"	
```

Comment:
Poedit would not add whitespace characters here.

Did you edit the file manually?

Reply:
You're right! I have manually edited the few first lines of the file while committing, that's my mistake didn't notice the whitespace.

Reply:
Can you open the file in Poedit and save it again? Would that remove the whitespace? If not, please fix the whitespace error.

Reply:
I've done as you suggested, the whitespace has been removed now.